man/tools.go

```diff
@@ -1,7 +0,0 @@
-//go:build tools
-
-package main
-
-import (
-	_ "github.com/cpuguy83/go-md2man/v2"
-)
```

Comment:
I removed the `tools.go` here, as we already vendor the dependency as indirect dependency of `docker/cli-docs-tool`. I could keep if it that's a concern (but things would show up pretty fast if we'd miss the dependency at some point)